### ISSUE TITLE
start ZIL o/c 10 sec before ZIL mining to let the memory crank up to the dead

### ISSUE THREAD

**Issue by slappy3** [Lolliedieb/lolMiner-releases#1894]
Is it possible to trigger the o/c setting 10 seconds before ZIL mining start especially the --zmoff. It take long to reach the max memory speed and we only have 1 minute of ZIL mining. When ZIL mining stop it can return to main algo o/c without delay.

**Comment by djRicky** [comment]
Good idea, or it will be better after the DAG has been created successfully.

**Comment by jgonzis** [comment]
It should all been applied, the problem is you see the 10 or 15s stats, that is why you think starts slower... but it is because count the time of the connection and the DAG creation
